src/core/loaders/Loader.ts

```diff
@@ -200,6 +202,27 @@ export abstract class Loader extends Disposable {
     return str
   }
 
+  private treeNodeValueHasPluralizationKeys(value: Record<string, any>, locale: string) {
+    if (!value)
+      return false
+
+    return isObject(value)
+      && this.pluralizationKeysV4.some((key: string) => get(value, key) != null)
+  }
+
+  private firstPluralizationKeyValue(value: Record<string, any>, locale: string) {
+    if (!value)
+      return undefined
+
+    for (const key of this.pluralizationKeysV4) {
+      const val = get(value, key)
+      if (!isObject(val) && val != null)
+        return val
+    }
+
+    return undefined
```

Comment:
You could possibly simplify this to one line:

```suggestion
    return value[Object.keys(value).find((key) => this.pluralizationKeysV4.includes(key))]
```

Or two lines to make it easier to read:
```
  const firstPluralizationKey = Object.keys(value).find((key) => this.pluralizationKeysV4.includes(key))
  return value[firstPluralizationKey]
```